docs/08-cicd.md

```diff
@@ -1,67 +1,93 @@
 # Continuous Integration and Delivery
+## Content
+* [Deliver a new release](#triggering-a-delivery-using-the-ci)
+* [Workflows](#workflows)
+* [Fastlane](#fastlane)
+* [Determining the next version](#determining-the-next-version)
+* [Environment variables](#environment-variables-and-dependencies)
+* [Used services](#services)
+* [Hints and quirks](#hints-and-quirks)
 
-# Workflows
+## Triggering a Delivery using the CI
 
-Several workflows exist for different purposes. See the table below for an overview.
+The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflow *api_triggered_delivery*.
 
-|Workflow|Type-checking|Jest tests|Build for E2E tests|E2E tests|Delivered to production|
-|---|---|---|---|---|---|
-|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|
-|weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|
-|bi-weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
-|api-triggered|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|-|
+* Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-pi-tokens/).
+* Trigger a build using the tool [trigger-pipeline.sh](../.circleci/trigger-pipeline.sh).
+* For more information on how to use it, execute it without parameters to see usage information.
 
-Whether `api-triggered` is delivered to production depends on the supplied parameters.
+## Workflows
 
-## About the "api-triggered" Workflow 
+Several workflows exist for different purposes:
+* **commit**: Run for all commit of a PR to ensure good code quality and working code.
+* **weekly_development_delivery**: Delivers 'integreat-test-cms' builds to development (Testflight and Beta channel) once a week.
+* **bi_weekly_production_delivery**: Delivers 'integreat' builds to production.
+* **api_triggered_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers either 'integreat-test-cms' builds to development
+ or 'integreat' builds to production.
 
-This workflow can be triggered using the tool `yarn trigger-pipeline` as described [below](#triggering-a-build-in-ci).
-Depending on the supplied parameters the build is released to production.
+See the table below for a more detailed overview:
 
-### Triggering a Build in CI
+|Workflow|Checks|E2E tests|Delivery|integreat build|integreat-test-cms build|malte build|
```

Comment:
Malte builds will be added with https://issues.integreat-app.de/browse/NATIVE-539